wordpress-mobile/WordPress-iOS
# Issues/7633 jetpack login

Closes #7633

This PR updates the Jetpack login screen shown in the Stats feature to use the new login flow.  The approach matches the current implementation in the Android app with two differences:
- The email screen does not provide an option to use the screen name instead.
![simulator screen shot - ipad air - 2017-12-16 at 14 05 30](https://user-images.githubusercontent.com/1435271/34073934-e0b8491e-e26a-11e7-8c2c-79058fbca46e.png)

- We continue to show a modified version of the current Jetpack prompt rather than show the login flow as soon as the stats menu item is selected.   This is both for a better experience on the iPad and to retain the informative and Jetpack version checking behavior.
![simulator screen shot - ipad air - 2017-12-16 at 14 05 24](https://user-images.githubusercontent.com/1435271/34073936-e5c3de1e-e26a-11e7-92b5-ee2d0d820a54.png)

To test:
- Start signed out
- Sign in to a self-hosted site. 
- View the stats feature and tap to install Jetpack
- Progress through the flow.  Confirm you are viewing the stats screen after dismissing the login epilogue.

Test on iPhone and iPad devices. 

Bonus: For paranoia, double check the version checking and more info features of the Jetpack login screen.

@koke would you be game for this one?
